Tests/SwiftStdlibTests/StringExtensionsTests.swift

```diff
@@ -627,10 +633,12 @@ final class StringExtensionsTests: XCTestCase {
     #if os(iOS)
     func testItalic() {
         let italicString = "hello".italic
+        // swiftlint:disable legacy_constructor
```

Comment:
Instead of disable all and enable all, it wouldn't be better just use //swiftlint:disable:next legancy_constructor ?

Reply:
The reason I'm doing this is due to the modular nature of extensions in SwifterSwift, so when users copy a single extension to their project where SwiftLint is used, it should just work

Reply:
Ahh I see, but what I mean was 
Instead of 
```swift
// swiftlint:disable legacy_constructor
let range = NSMakeRange((text as NSString).length - 1, 1)
scrollRangeToVisible(range)
// swiftlint:enable legacy_constructor
```
Just to use 
```swift
// swiftlint:disable:next legacy_constructor
let range = NSMakeRange((text as NSString).length - 1, 1)
scrollRangeToVisible(range)
```
To disable the rule only for the next time, so it doesn't have disable and  then enable, just one command :))  But it's ok this way 👍

Reply:
Hmmm, now I understand, definitely it makes more sense, I'll update it throughout the project

Reply:
Didn't know this syntax exists 💯